src/msgpack/core.clj

```diff
@@ -5,6 +5,10 @@
            java.io.ByteArrayInputStream
            java.nio.charset.Charset))
 
+(def ^:private
+  default-charset
```

Comment:
This shouldn't be configurable, so I'd like to avoid using "default" in the name. Can we call this `msgpack-charset` instead? Since UTF-8 is specifically required by the MessagePack spec.